public/convert.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
I think this file may be leftover from the TrialScope implementation

Reply:
This file is definitely left over from TrialScope, it was used to test the convert condition code to TrialScope code service.